primitives/chain-bridge-hub-cumulus/src/lib.rs

```diff
@@ -134,88 +133,8 @@ pub const MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX: MessageNonce = 1024;
 /// Maximal number of unconfirmed messages at inbound lane for Cumulus-based parachains.
 pub const MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX: MessageNonce = 4096;
 
-/// Extra signed extension data that is used by all bridge hubs.
-pub type SignedExtra = (
-	CheckNonZeroSender,
-	CheckSpecVersion,
-	CheckTxVersion,
-	CheckGenesis<Hash>,
-	CheckEra<Hash>,
-	CheckNonce<Nonce>,
-	CheckWeight,
-	ChargeTransactionPayment<Balance>,
+/// Signed extension that is used by all bridge hubs.
+pub type SignedExtension = SuffixedCommonSignedExtension<(
 	BridgeRejectObsoleteHeadersAndMessages,
 	RefundBridgedParachainMessagesSchema,
-);
-
-/// Signed extension that is used by all bridge hubs.
-pub type SignedExtension = GenericSignedExtension<SignedExtra>;
-
-/// Helper trait to define some extra methods on bridge hubs signed extension (and
-/// overcome Rust limitations).
-pub trait BridgeHubSignedExtension {
```

Comment:
Once you merge this, please also update `polkadot-sdk` and fix breaking changes like this one while still fresh in our minds 😄 . (used [here](https://github.com/paritytech/polkadot-sdk/blob/bfb241d7f3e29152b9fe0ab3a0364ee0c89ec65c/cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs#L1269))

Reply:
Ok, I will. I have to merge it to polkadot-staging first. And then will try to update the bridges repo in polkadot-sdk.